### DISCUSSION TITLE
All buttons not working (fixed in >v1.23.10)

### DISCUSSION THREAD

**Discussion by denisteixeira** [pglombardo/PasswordPusher#660] [General]
Hi everyone,

After configure/install the pwdpusher on kubernetes cluster my page won´t let me click on any button.

Did you ever get this error after install on kubernetes?

Thanks

![image](https://user-images.githubusercontent.com/114034905/212179241-944014c8-929c-49df-91d4-f1d3307373a8.png)

**Comment by pglombardo** [comment]
This might be due to recent changes.  The `release` docker tag is still stable.  

Two questions:

1. What tag did you deploy?
2. What's in the browser javascript console?  (Right click page, inspect, console tab)

**Comment by denisteixeira** [comment]
Hi pglombardo,


i'm using master tag. (i think this set the lastest version: image: docker.io/pglombardo/pwpush-ephemeral)

on browser inspection (in console tab) i received this error:

![image](https://user-images.githubusercontent.com/114034905/212190393-d7a12d86-dd2c-4beb-999a-7b876a2fa1cd.png)

but if i'm use the port forward on kubernetes these buttons won´t work too.

did i forget something?

Thanks man

**Reply by pglombardo** [reply]
Hi @denisteixeira - [v1.23.10](https://github.com/pglombardo/PasswordPusher/releases/tag/v1.23.10) which is building now should resolve this.  Could you try it out and let me know.  The Docker build usually takes about 45 minutes.

**Comment by denisteixeira** [comment]
Thanks @pglombardo,

This tag solve the problem, thank you !!!! Have a good one!

![image](https://user-images.githubusercontent.com/114034905/212332753-90c2bbb5-477a-4cc4-b035-a30e82cdfdfb.png)
